Automattic/wp-calypso
# No sites illustration: import it dynamically instead of relying on the URL

Follows up https://github.com/Automattic/wp-calypso/pull/82217.

## Proposed Changes

The production version of Calypso is still showing the old version of the SVG because of the name of the file. As the static folder is in discouraged, I moved the image it to assets folder and imported it through ESModules so that it generates a new file name on every change to the file.

| Before | After |
| -------| ------|
| <img width="1438" alt="image" src="https://github.com/Automattic/wp-calypso/assets/26530524/c859cb7b-3a7d-45f0-8e30-868eabe30e93"> | <img width="1438" alt="image" src="https://github.com/Automattic/wp-calypso/assets/26530524/ddfe5504-a0bb-4d5d-8983-f16cabc4a884"> |

## Testing Instructions

Browse `/sites` and verify that the no sites placeholder contains the new graphic instead of the old one.
